src/styles/blocks/calendar.scss

```diff
@@ -24,14 +30,16 @@
 
     font-family: Arial, Helvetica, sans-serif;
     font-size: 30px;
-    transition: transform 0.5s;
 
     border: 1px solid #000;
     background: #eee;
 
+    transition-duration: 0.5s;
```

Comment:
look at your Demo. Now everything that happens with days block has a duration of the animation.
You shouldn't delete transform, but add color and etc.)